sentinel-transport/sentinel-transport-spring-mvc/src/main/java/com/alibaba/csp/sentinel/transport/heartbeat/SpringMvcHttpHeartbeatSender.java

```diff
@@ -101,8 +101,7 @@ public boolean sendHeartbeat() throws Exception {
         if (statusCode == OK_STATUS) {
             return true;
         } else if (clientErrorCode(statusCode) || serverErrorCode(statusCode)) {
-            RecordLog.warn("[HttpHeartbeatSender] Failed to send heartbeat to "
-                + consoleHost + ":" + consolePort + ", http status code: " + statusCode);
+            RecordLog.warn("[HttpHeartbeatSender] Failed to send heartbeat to {}:{}, http status code: {}" , consoleHost,consolePort,statusCode);
```

Comment:
the same like above. and keep a ` ` after `,`

Reply:
done